### PR TITLE
[Communication] - Phone Numbers - Address Api View Comments

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

**Review by jbeauregardb — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java [hunk 1]
```diff
@@ -179,7 +180,8 @@
                 .flatMap((PhoneNumbersSearchAvailablePhoneNumbersResponse response) -> {
                     pollingContext.setData("operationId", response.getDeserializedHeaders().getOperationId());
                     pollingContext.setData("searchId", response.getDeserializedHeaders().getSearchId());
-                    return client.getOperationAsync(pollingContext.getData("operationId"));
+                    Mono<PhoneNumberOperation> operation = getOperation(pollingContext.getData("operationId"));
+                    return operation;
```
Comment (beltr0n) [inline sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java:184]: nit: any particular reason why this is being assigned to local variable before being returned?

**Review by beltr0n — COMMENTED** [review]
one nit comment, but otherwise looks good :)

**Review by JoshuaLai — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java:184]: Oh grreat catch! I was doing some testing and totally forgot to revert this. Thank you, I'll fix

Comment (beltr0n) [reply sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java:184]: i figured but didn't want to assume :)

**Review by beltr0n — APPROVED** [review]

**Review by DominikMe — APPROVED** [review]

**Pull request merged**